keras_cv/models/backbones/efficientnet_v1/efficientnet_v1_backbone.py

```diff
@@ -0,0 +1,476 @@
+        width_coefficient: float, scaling coefficient for network width.
+        depth_coefficient: float, scaling coefficient for network depth.
+        dropout_rate: float, dropout rate before final classifier layer.
+        drop_connect_rate: float, dropout rate at skip connections.
+        depth_divisor: integer, a unit of network width.
+        activation: activation function to use between each convolutional layer.
+        input_shape: optional shape tuple, it should have exactly 3 input
+            channels.
+        input_tensor: optional Keras tensor (i.e. output of `layers.Input()`) to
+            use as image input for the model.
+        stackwise_kernel_sizes:  list of ints, the kernel sizes used for each
+            conv block.
+        stackwise_num_repeats: list of ints, number of times to repeat each
+            conv block.
+        stackwise_input_filters: list of ints, number of input filters for
+            each conv block.
+        stackwise_output_filters: list of ints, number of output filters for
+            each stack in the conv blocks model.
+        stackwise_expansion_ratios: list of floats, expand ratio passed to the
+            squeeze and excitation blocks.
+        stackwise_strides: list of ints, stackwise_strides for each conv block.
+        stackwise_squeeze_and_excite_ratios: list of ints, the squeeze and
+            excite ratios passed to the squeeze and excitation blocks.
+
+    Usage:
+    ```python
+    # Construct an EfficientNetV1 from a preset:
+    efficientnet = keras_cv.models.EfficientNetV1Backbone.from_preset(
+        "efficientnetv1_b0"
+    )
+    images = tf.ones((1, 256, 256, 3))
+    outputs = efficientnet.predict(images)
+
+    # Alternatively, you can also customize the EfficientNetV1 architecture:
+    model = EfficientNetV1Backbone(
+        stackwise_kernel_sizes=[3, 3, 5, 3, 5, 5, 3],
+        stackwise_num_repeats=[1, 2, 2, 3, 3, 4, 1],
+        stackwise_input_filters=[32, 16, 24, 40, 80, 112, 192],
+        stackwise_output_filters=[16, 24, 40, 80, 112, 192, 320],
+        stackwise_expansion_ratios=[1, 6, 6, 6, 6, 6, 6],
+        stackwise_strides=[1, 2, 2, 2, 1, 2, 1],
+        stackwise_squeeze_and_excite_ratios=[
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+        ],
+        width_coefficient=1.0,
+        depth_coefficient=1.0,
+        include_rescaling=False,
+    )
+    images = tf.ones((1, 256, 256, 3))
+    outputs = efficientnet.predict(images)
+    ```
+    """  # noqa: E501
+
+    def __init__(
+        self,
+        *,
+        include_rescaling,
+        width_coefficient,
+        depth_coefficient,
+        dropout_rate=0.2,
+        drop_connect_rate=0.2,
+        depth_divisor=8,
+        activation="swish",
+        input_shape=(None, None, 3),
+        input_tensor=None,
+        stackwise_kernel_sizes,
```

Comment:
don't args with defaults always need to come after args without them?

Regardless, let's put these stackwise args first to match other backbones.